R/nonmem.R

```diff
@@ -859,6 +892,8 @@ rxToNonmem <- function(x, ui) {
           }
         }
       }
+    } else {
```

Comment:
When does this happen?  Can you trigger it. 

The error message is a bit confusing to me.

Reply:
It shouldn't ever happen.  This is intended to be a catch-all in case something is missed.